ng/src/web/pages/groups/component.js

```diff
@@ -39,36 +40,42 @@ class GroupComponent extends React.Component {
   constructor(...args) {
     super(...args);
 
+    this.state = {dialogVisible: false};
+
+    this.closeGroupDialog = this.closeGroupDialog.bind(this);
     this.openGroupDialog = this.openGroupDialog.bind(this);
   }
 
   openGroupDialog(group) {
     const {gmp} = this.props;
 
+    let allUsers = [];
+    gmp.users.getAll().then(response => {
+      allUsers = response.data;
```

Comment:
You have to put allUsers into the state here otherwise it is currently always an empty array.